src/matcher/commonjs.js

```diff
@@ -2,13 +2,13 @@
 
 var regexToArray = require('../util/regex-to-array');
 
-function Matcher() {}
-Matcher.prototype = {
-  match: function (code) {
+module.exports = function ($fs) {
+  return function (file, code) {
     return regexToArray(/require\([\'"]([^\'"]+)[\'"]\)/g, code).map(function (imp) {
```

Comment:
I guess all these matchers aren't aware about whether you're inside a string that happens to have contents that look like an import. Definitely not suggesting to build an AST out of every file, but just to be aware of a possible failure case.